app/models/company.rb

```diff
@@ -17,4 +17,5 @@ def d3_coordinates
     slice(:longitude, :latitude)
   end
 
+  STATUS_OPTIONS = { "Uncontacted" => "uncontacted", "Contacted" => "contacted", "Confirmed" => "confirmed", "Dead" => "dead" }
```

Comment:
We might be able to do a before_save and then downcase the status at that point? This works too though.